apis/nucleus/src/components/listbox/interactions/keyboard-navigation/keyboard-nav-rows.js

```diff
@@ -119,9 +119,8 @@ export default function getRowsKeyboardNavigation({
         }
         break;
       case KEYS.ESCAPE:
-        if (isModal()) {
-          cancel();
-        } else {
+        cancel();
+        if (typeof isModal === 'undefined') {
```

Comment:
I would expect this to break escape in filterpane where it expect it to propagate only if `isModal() returns false`
have you tested the change there?

Reply:
Yes I had tested it with filterpane but I think I found the case you concerned and I hadn't tried. When there is a text in the filterpane searchbox isModal() returns false so I turned that condition back.